### PR TITLE
Gowtham

### PR DESCRIPTION
**Pull request by GowthamReddy-E**
this file has some git commands .if any commands missing please add it and rise pr

### CODE REVIEW

**Review by Narendram22 — APPROVED** [review]

**Review by manjulakusuma — APPROVED** [review]

File: k8s commands.txt
Comment (Mahammad046) [inline k8s commands.txt:1]: ok

**Review by Mahammad046 — APPROVED** [review]
ok

**Review by omnath1234 — APPROVED** [review]

**Comment by reddyjalla** [comment]
wow

**Comment by Jaggu444** [comment]
Ok

**Review by Jaggu444 — APPROVED** [review]

**Pull request closed without merging**